omniscidb/ArrowStorage/ArrowStorage.h

```diff
@@ -73,7 +73,8 @@ class ArrowStorage : public SimpleSchemaProvider, public AbstractDataProvider {
   std::unique_ptr<Data_Namespace::AbstractDataToken> getZeroCopyColumnData(
       const ColumnRef& col_ref) override;
 
-  TableFragmentsInfo getTableMetadata(int db_id, int table_id) const override;
+  const std::shared_ptr<TableFragmentsInfo> getTableMetadata(int db_id,
```

Comment:
What is this `const` for here and all other places? Did you mean to have `std::shared_ptr<const TableFragmentsInfo>`?